doc/develop/toolchains/crosstool_ng.rst

```diff
@@ -1,5 +1,13 @@
-Crosstool-NG
-############
+.. _toolchain_xtools:
+
+Crosstool-NG (Deprecated)
+#########################
+
+.. note::
```

Comment:
.. warning:: would be preferred IMO to make the information more prominent

Reply:
Changed to warning.